MANIFEST.in

```diff
@@ -1 +1,2 @@
 include *.txt
+include tests.py
```

Comment:
we should not ship tests.py in the package

Reply:
Nowadays it's recommended to ship tests too. For example, downstream packagers like Fedora want to run the tests to make sure it's working.